radio/test_sessions/tasklist_features.py

```diff
@@ -0,0 +1,11 @@
+tasks = [
+    [tc.SendBeacon(), SendLoop(2), WaitMode.NoWait],
+    [tc.SendBeacon(), SendReceive, WaitMode.NoWait],
+    [tc.SendBeacon(), Print("It shows how the Print() action works. It does not send any telecommand. It just prints a message."), WaitMode.NoWait],

```

Comment:
Szczerze to mi sie nie podoba - nie jest to oczywiste co ten `Print` robi jako drugi argument, po telekomendzie. Nie jest oczywiste jaka jest relacja między `Print` a `tc.SendBeacon()`. IMO to wnosi niepotrzebne zamieszanie. 

Lepiej by było gdyby `Print` był poprostu kolejnym elementem na taskliście, np. z "pustą" telekomendą (np.: noop).

Reply:
Dokładnie tak myślałem, zeby to był NOP i się zgadzam. Nie chciałem się w to bawić, bo wydawało mi się za dużo roboty i zostawiłem jak jest. Looknę czy może da się to prosto zrobić.

Reply:
Fixed :white_check_mark: